diff --git a/node_modules/react-native-actions-sheet/dist/src/index.js b/node_modules/react-native-actions-sheet/dist/src/index.js
index a5a1305..e0c93dd 100644
--- a/node_modules/react-native-actions-sheet/dist/src/index.js
+++ b/node_modules/react-native-actions-sheet/dist/src/index.js
@@ -122,9 +122,10 @@ export default forwardRef(function ActionSheet(_a, ref) {
     var closing = useRef(false);
     var draggableNodes = useRef([]);
     var layoutTimeoutRef = useRef(null);
+    var _windowDims = Dimensions.get('window');
     var _2 = useState({
-        width: -1,
-        height: -1,
+        width: _windowDims.width,
+        height: _windowDims.height,
     }), dimensions = _2[0], setDimensions = _2[1];
     var dimensionsRef = useRef(dimensions);
     dimensionsRef.current = dimensions;
@@ -1042,7 +1043,9 @@ export default forwardRef(function ActionSheet(_a, ref) {
                     return;
                 setDimensions({
                     width: event.nativeEvent.layout.width,
-                    height: event.nativeEvent.layout.height,
+                    height: Platform.OS === 'android'
+                       ? Dimensions.get('window').height
+                        : event.nativeEvent.layout.height,
                 });
             }} ref={rootViewContainerRef} pointerEvents={(props === null || props === void 0 ? void 0 : props.backgroundInteractionEnabled) ? 'box-none' : 'auto'} style={[
                 styles.parentContainer,
